src/modules/uv-searchfooterpanel-module/FooterPanel.ts

```diff
@@ -623,27 +623,11 @@ export class FooterPanel extends BaseFooterPanel {
     }
 
     setPlacemarkerLabel(): void {
-
         const displaying: string = this.content.displaying;
         const index: number = this.extension.helper.canvasIndex;
+        const prefix: string = this.isPageModeEnabled ? this.content.page : this.content.image;
 
-        if (this.isPageModeEnabled()) {
-            const canvas: Manifesto.ICanvas = this.extension.helper.getCanvasByIndex(index);
-            let label: string | null = Manifesto.TranslationCollection.getValue(canvas.getLabel());
-
-            if (!label) {
-                label = this.content.defaultLabel;
-            }
-
-            const lastCanvasOrderLabel: string | null = this.extension.helper.getLastCanvasLabel(true);
-
-            if (lastCanvasOrderLabel) {
-                this.$pagePositionLabel.html(String.format(displaying, this.content.page, UVUtils.sanitize(<string>label), UVUtils.sanitize(<string>lastCanvasOrderLabel)));
-            }
-
-        } else {
-            this.$pagePositionLabel.html(String.format(displaying, this.content.image, index + 1, this.extension.helper.getTotalCanvases()));
-        }
+        this.$pagePositionLabel.html(String.format(displaying, prefix, index + 1, this.extension.helper.getTotalCanvases()));
```

Comment:
I'm kinda surprised to see this change in our local repository.

Just reading over the code really quickly, would it be possible to set `isPageModeEnabled` to false in the configuration so we end up in the correct branch and not have to maintain this customization?  (Perhaps I'm not noticing something else that's going on here)

Reply:
Sadly not. The `isPageModeEnabled` is calculated from our manifests, which have `paged` as the viewing hint in them. This is intended behaviour on our part, since we want those items to display as books. UV should show two pages at once in these cases, with a number of other panels using this flag to adjust their layout to better reflect paged objects.

I agree it's undesirable to delete a branch of the logic unnecessarily. In this case, the actual differences between the page and image display labels were not implemented correctly, leading to weird labels. The assumption was that text labels in the manifest would always be of the form "Page 1, Page2" etc., similar to the way PDFs have a "page" number and a "this is the number of digital rectangles his PDF has" number. This assumption can't be counted on given the spec, and the fallback behaviour was what we were seeing as an error in this issue.

Reply:
So is this something that we could fix upstream in UV itself, or would there be some better way to represent the labels in our manifests?

Reply:
Yes, ideally we'd fix it upstream rather than maintaining the fork. Mark wanted to hold off on submitting that in a PR until there was a chance to discuss the issue for some reason. Here's the upstream issue: https://github.com/UniversalViewer/universalviewer/issues/531